include/aos/common/config.hpp

```diff
@@ -155,4 +155,88 @@
 #define AOS_CONFIG_MONITORING_POLL_PERIOD_SEC 10
 #endif
 
+/**
+ * Certificate public key len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_PUB_KEY_LEN
+#define AOS_CONFIG_CRYPTO_CERT_PUB_KEY_LEN 2048
+#endif
+
+/**
+ * Certificate private key len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_PRIV_KEY_LEN
+#define AOS_CONFIG_CRYPTO_CERT_PRIV_KEY_LEN 2048
+#endif
+
+/**
+ * Certificate serial number len.
+ */
+#ifndef AOS_CONFIG_CRYPTO_CERT_SERIAL_LEN
+#define AOS_CONFIG_CRYPTO_CERT_SERIAL_LEN 40
```

Comment:
Looks like it should be 20 byte len: https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.2